06.api-reference/28.picks/docs.md

```diff
@@ -124,53 +122,53 @@ Adds a new top-level menu.
 |The ID of the pick created|number|Used for managing the Pick. `0` if invalid. |
 
 
-### `disablePick(uid)`<a id="m2"></a>
+### `disablePick(id)`<a id="disablePick"></a>
```

Comment:
Uuid = universally unique id (128 bits randomly selected --> the odds of a collision is so small we ignore it)

id = 32-bit integer id.  It is unique within its context, but not universally so.